push/mv.go

```diff
@@ -207,6 +208,21 @@ func (m *MaterializedView) Fill(shardIDs []uint64, interruptor *interruptor.Inte
 		return errors.WithStack(err)
 	}
 
+	// Start any sources that wait to ingest on first MV
+	for _, tableExec := range tes {
+		if tableExec.IngestOnFirstMV() {
+			source, err := m.pe.GetSource(tableExec.TableInfo.ID)
+			if err != nil {
+				return errors.WithStack(err)
+			}
+			if !source.IsRunning() {
```

Comment:
Looks like a possible race condition here. If you change source Start() to not do anything if already running that will fix this.

Reply:
Done. Looks like if you run source.Start() and it's already started, it already doesn't do anything. Should've looked before writing this check